Makefile

```diff
@@ -151,7 +151,7 @@ ifndef HAS_GINKGO
 endif
 
 build-vendor:
-	${DEV_ENV_CMD} dep ensure
+	${DEV_ENV_CMD} dep ensure --vendor-only
```

Comment:
I think we'd want `--vendor-only`  to remain disabled for our current vendoring and vendor validations.